Add Ceres products to SORA docs (#225)
sora-xor/sora-docs: src/ceres/overview.md

---
title: "Overview | SORA Docs"
head:
  - - meta
    - name: description
      content: "Ceres is developing DeFi services and utilities in DotSama ecosystem."
  - - meta
    - name: keywords
      content: "Ceres, DeFi, liquidity locker, airdrops, demeter, hermes, apollo, DotSama, Polkaswap, SORA network"
---

## Ceres Ecosystem

> Ceres is one of the first DeFi service providers in the DotSama ecosystem and the first one on SORA network.

Reviewer: ```suggestion
Ceres is one of the first DeFi service providers in the DotSama ecosystem and the first on the SORA network.
```

> Ceres provides DeFi services and utilities for the new projects and tokens in the DotSama ecosystem.<br>

Reviewer: ```suggestion
Ceres provides DeFi services and utilities for new projects and tokens in the DotSama ecosystem.<br>
```

> CERES is a native token of the Ceres ecosystem, where the concepts of burning, airdrops, treasury and governance are in one place.

Reviewer: ```suggestion
CERES is the native token of the Ceres ecosystem, where the concepts of burning, airdrops, treasury, and community governance are all in one place.
```

## CERES token utility

- Governance - CERES token holders can influence decisions concerning the project, either by making proposals or deciding on other’s proposals for new features.
- Airdrops - The goddess would always generously reward her admirers. Every CERES holder can get each of three Ceres Airdrop tokens (Demeter, Apollo, and Hermes). Each airdrop will have its purpose and will be provided with initial liquidity.
- Treasury - Part of the fees from all platforms in the Ceres ecosystem will be transferred to a public treasury wallet managed by CERES holders using the governance platform.

> - Deflationary - For the goddess to grant all the wishes of her followers, it is necessary to offer an adequate sacrifice. CERES tokens will be burned over time and the supply will constantly be reducing.

Reviewer: ```suggestion
- Deflationary - For the goddess to grant all the wishes of her followers, it is necessary to offer an adequate sacrifice. CERES tokens will be burned over time and the supply will be constantly reducing.
```

<br>

**CERES tokenomics**

> Team - 3,750 <br>

Reviewer 2: Please provide a name for the token

Reviewer: the token is called CERES

---
title: "Overview | SORA Docs"
head:
  - - meta
    - name: description
      content: "Ceres is developing DeFi services and utilities in DotSama ecosystem."
  - - meta
    - name: keywords
      content: "Ceres, DeFi, liquidity locker, airdrops, demeter, hermes, apollo, DotSama, Polkaswap, SORA network"
---

> ## Ceres Ecosystem

Reviewer: ```suggestion
# Ceres Ecosystem Overview
```
